include/picongpu/param/transitionRadiation.param

```diff
@@ -233,20 +240,23 @@ namespace listFrequencies
          */
         /** get index for computing angle theta: */
         const int indexTheta = observation_id_extern / parameters::nPhi;
+
+        /** step width angle theta */
+        const picongpu::float_64 deltaTheta = ( parameters::nTheta > 1 ) ?
+                ( parameters::thetaMax - parameters::thetaMin ) / ( parameters::nTheta - 1.0 ) : 0.0;
+
+        /** compute observation angles theta */
+        const picongpu::float_64 theta = indexTheta * deltaTheta + parameters::thetaMin;
+
         /** get index for computing angle phi: */
         const int indexPhi = observation_id_extern % parameters::nPhi;
 
-        /** step width angle theta */
-        const picongpu::float_64 deltaTheta = ( parameters::thetaMax -
-            parameters::thetaMin ) / ( parameters::nTheta - 1.0 );
         /** step width angle phi */
-        const picongpu::float_64 deltaPhi = ( parameters::phiMax -
-            parameters::phiMin ) / ( parameters::nPhi - 1.0 );
+        const picongpu::float_64 deltaPhi = ( parameteters::nPhi > 1 ) ?
```

Comment:
Typo in `parameters`. (The tests failed because of it). Otherwise I like your changes, thanks!